README.md

```diff
@@ -13,7 +13,7 @@ git clone https://github.com/biocore/metagenomics_pooling_notebook.git
 Create a Python3 Conda environment in which to run the notebook:
 
 ```bash
-conda create -n metapool 'python=3.9' scikit-learn pandas numpy scipy nose pep8 flake8 matplotlib jupyter notebook 'seaborn>=0.7.1' pip openpyxl
+conda create -n metapool 'python=3.9' scikit-learn pandas numpy scipy nose pep8 flake8 matplotlib jupyter 'notebook==6.5.5' 'seaborn>=0.7.1' pip openpyxl
```

Comment:
why this pin?

Reply:
The following commands in the README can't be executed if notebook is 7.0.0 or higher:
jupyter contrib nbextension install --sys-prefix --skip-running-check
jupyter nbextension enable toc2/main

I haven't looked at the changes in 7.0.0 so there might be a better way of handling this. The pin was mostly to remind me not to let it slip under the radar.

Reply:
I think is worth spending the time to see what changed and decide if needs to be fixed; could you confirm what changed?

Reply:
Will do! ty!

Reply:
@antgonza Resolved. It turns out the only extension used is the table-of-contents extension, which is a new built-in feature of version 7.0.0. I modified the README.md so that only users of older notebook versions will need to perform these steps.